ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_l3_edge_l3_interfaces.schema.yml

```diff
@@ -52,6 +52,43 @@ $defs:
         qos_profile:
           type: str
           description: QOS service profile.
+        wan_path_group:
+          type: str
+          documentation_options:
+            table: wan-interfaces-settings
+          description: |-
+            PREVIEW: This key is currently not supported
+
+            The WAN path-group this interface is connected to.
+        wan_carrier:
+          type: str
+          documentation_options:
+            table: wan-interfaces-settings
+          description: |-
+            PREVIEW: This key is currently not supported
+
+            The WAN Carrier this interface is connected to.
+            This is not rendered in the configuration but used for WAN designs.
+        wan_circuit_id:
+          type: str
+          documentation_options:
+            table: wan-interfaces-settings
+          convert_types:
+            - int
+          description: |-
+            PREVIEW: This key is currently not supported
+
+            The WAN Circuit ID for this interface.
+            This is not rendered in the configuration but used for WAN designs.
+        cv_pathfinder_connected_to_pathfinder:
+          type: bool
+          documentation_options:
+            table: wan-interfaces-settings
+          description: |-
+            PREVIEW: This key is currently not supported
+
+            For a WAN interface (`wan_carrier` is set), allow to disable the static tunnel towards Pathfinders.
```

Comment:
`wan_carrier` is set -> Is this wan_carrier or pathgroup? Or is it always required that both wan_carrier and path_group be specified?

Reply:
good catch - too many renaming - should be `wan_path_group`